benchopt/cli/main.py

```diff
@@ -62,6 +62,8 @@
               help='Timeout a solver when run for more than <timeout> seconds')
 @click.option('--plot/--no-plot', default=True,
               help="Whether or not to plot the results. Default is True.")
+@click.option('--html/--no-html', default=True,
+              help="Whether to display the plot as HTML report or matplotlib figures, default is True.")
```

Comment:
Careful that we ask the lines to be less than 80 characters

```suggestion
              help="Whether to display the plot as HTML report or matplotlib figures, "
              "default is HTML.")
```